src/content/docs/new-relic-solutions/observability-maturity/customer-experience/quality-foundation-implementation-guide.mdx

```diff
@@ -203,7 +203,7 @@ For each KPI, we defined thresholds - one for warning, another for critical.  Yo
 
 Quality Foundation helps you identify where in your application you need to make improvements that will optimize user retention, conversion and satisfaction. It is less about where things are and more about where to get to.
 
-It also shows you what you should be measuring going forward. You can use this to define SLOs (in a service level dashboard) and alert on them. 
+It also shows you what you should be measuring going forward. You can use this to define [SLOs](/docs/service-level-management/intro-slm/) (in a service level dashboard) and alert on them. 
```

Comment:
```suggestion
It also shows you what you should be measuring going forward. You can use this to define [service-level objectives (SLOs)](/docs/service-level-management/intro-slm/) (in a service level dashboard) and alert on them. 
```

How about we make this acronym even more friendly by exploding it @amyreinholds?

Reply:
:)